### PR TITLE
feat: disable spiltChunks at dev to prevent preload error

### PR DESCRIPTION
**Pull request by Repraance**
**What kind of change does this PR introduce?**

**Did you add tests(unit,e2e) for your changes?**

**Does this PR introduce a breaking change?**

**What needs to be documented once your changes are merged?**

### CODE REVIEW

File: packages/platform-web/src/shuvi-app/app/client.ts [hunk 1]
```diff
@@ -141,7 +141,8 @@ export const createApp: CreateAppClient = ({
         next();
         return;
       }
-
+      // should show unexpected error
+      console.error(error);
```
Comment (liximomo) [inline packages/platform-web/src/shuvi-app/app/client.ts:145]: 这个需要加吗？我们 error-over-lay 有展示error的吧，如果没展示，那就是个bug，请报给 harry，并告诉他怎么复现

File: packages/toolpack/src/webpack/config/base.ts [hunk 2]
```diff
@@ -131,24 +130,7 @@ export function baseWebpackChain({
     realContentHash: false
   });
 
-  if (dev) {
-    config.optimization.usedExports(false);
```
Comment (liximomo) [inline packages/toolpack/src/webpack/config/base.ts:135]: 这行为什么需要移除？

**Review by liximomo — CHANGES_REQUESTED** [review]

File: packages/toolpack/src/webpack/config/base.ts [hunk 3]
```diff
@@ -131,24 +130,7 @@ export function baseWebpackChain({
     realContentHash: false
```
Comment (liximomo) [inline packages/toolpack/src/webpack/config/base.ts:130]: 这里需要加一行 splitChunks: false，禁掉默认的 splitChunks 逻辑

https://github.com/shuvijs/shuvi/blob/v1.0.0-rc.1/packages/toolpack/src/webpack/config/base.ts#L92

**Review by liximomo — CHANGES_REQUESTED** [review]

**Comment by liximomo** [comment]
又一个 e2e 挂了

File: packages/toolpack/src/webpack/config/base.ts (hunk 3 above)
Comment (Repraance) [reply packages/toolpack/src/webpack/config/base.ts:131]: fixed

File: packages/toolpack/src/webpack/config/base.ts (hunk 2 above)
Comment (Repraance) [reply packages/toolpack/src/webpack/config/base.ts:135]: 误删 fixed

File: packages/platform-web/src/shuvi-app/app/client.ts (hunk 1 above)
Comment (Repraance) [reply packages/platform-web/src/shuvi-app/app/client.ts]: fixed

**Review by liximomo — APPROVED** [review]
/lgtm

**Pull request merged**